### PR TITLE
Fixing command typo - Closes #274

### LINKED ISSUE #274: [doc] 5.5.1  MDS Tuning - Typo: ceph-daemon perf dump -> ceph daemon perf dump
5.5.1  MDS Tuning

https://documentation.suse.com/ses/6/single-html/ses-tuning/#id-1.5.3.4.12.3

---------------------
I assume

ceph-daemon perf dump

should read

ceph daemon perf dump

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/tuning-ceph-tuning.xml [hunk 1]
```diff
@@ -415,7 +415,7 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
        utilizing very low-latency media, such as NVMe, for the cephfs metadata pool.
      </para>
      <para>
-       Utlizing the <command>ceph-daemon perf dump</command> command, there is a significant
+       Utlizing the <command>ceph daemon perf dump</command> command, there is a significant
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:418]:
```suggestion
       Utilizing the <command>ceph daemon perf dump</command> command, there is a significant
```

File: xml/tuning-ceph-tuning.xml [hunk 2]
```diff
@@ -415,7 +415,7 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
...
        amount of data that can be examined for the &ceph; Metadata Servers.
        It should be noted that the MDS perf counters only apply to metadata
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:420]:
```suggestion
       It should be noted that the MDS performance counters only apply to metadata
```

File: xml/tuning-ceph-tuning.xml (hunk 1 above)
Comment (asettle) [reply xml/tuning-ceph-tuning.xml:418]: Personally, I'm a bit fan of utlizing but suit yourself

File: xml/tuning-ceph-tuning.xml [hunk 3]
```diff
@@ -415,7 +415,7 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
...
        operations. The actual IO path is from clients straight to OSDs.
```
Comment (lproven) [inline xml/tuning-ceph-tuning.xml:421]:
```suggestion
       operations. The actual I/O path is from clients straight to OSDs.
```

**Pull request merged**